applications/dashboard/scss/editor/_richEditor.scss

```diff
@@ -77,7 +77,7 @@
     flex-wrap: nowrap;
     list-style: none;
     padding: 0;
-    width: calc(100% - #{2 * $richEditor-border_width});
+    //width: calc(100% - #{2 * $richEditor-border_width});
```

Comment:
What's up with this commented line?

Reply:
I'm guessing that's also the reason for the whacky font size, but it does cause the menu bars to be incorrect widths so things need to change there.